gap/matrix/classical.gi

```diff
@@ -1712,11 +1723,25 @@ RECOG.NonGenericOrthogonalPlus := function(recognise,grp)
              return false;
         fi;
     elif d = 8 and  q =  5 then
-        #ACN Feb 07: added fix - need also elements of order a mult of 3
-        #            these are missing in the paper [NP99]
-        if not HasElementsMultipleOf( recognise.orders, [7,13,3])  then
+        ## 2.July.2019: There is a mistake in the paper here
+        ## There are maximal subgroups of Omega+(8,5) that contain 
+        ## elements of order 7,13,3
+        if not HasElementsMultipleOf( recognise.orders, [7,13,3,312])  then
```

Comment:
So where does the 312 come from? It's not a prime, and I just had a look at POmega(8,5) and based on a random sampling it only contains elements of order 156, not 312. 

Wouldn't 31 make more sense?